### ISSUE TITLE
Re-query is triggered unexpectedly when row from grid is selected and map moves

### ISSUE THREAD

**Issue by rladines** [Chicago/opengrid#268]
This was a side effect of fixing the issue identified by @reve0716 (not in repo?) where re-query is not triggered when moving the map using the keyboard.

**Comment by rladines** [comment]
Ready for re-test in INT/QA

**Comment by PriyaDoIT** [comment]
tested in INT -
1. map no longer refreshes when a row in the grid is selected
2. i was able to move the map using keyboard and the map refreshed for the new map extent

pass.

**Comment by reve0716** [comment]
I was able to test the mobility of map grid when selecting a column on table grid.  It no longer moves. But I am getting an error when I try to run all data-sets and select a table column. The query times out before all data-set have been executed with a message.  Tested in Both firefox and chrome.

![image](https://cloud.githubusercontent.com/assets/12734080/20320008/ee725094-ab35-11e6-97f2-b584552c5791.png)
